terraform/modules/jaspersoft/box.tf

```diff
@@ -0,0 +1,62 @@
+resource "aws_security_group" "jaspersoft_server" {
+  vpc_id      = var.vpc_id
+  description = "Jaspersoft server instance"
+}
+
+# tfsec:ignore:aws-vpc-no-public-egress-sgr
+resource "aws_security_group_rule" "jaspersoft_server_egress" {
+  type              = "egress"
+  description       = "Allow all egress"
+  from_port         = 0
+  to_port           = 0
+  protocol          = -1
+  cidr_blocks       = ["0.0.0.0/0"]
+  security_group_id = aws_security_group.jaspersoft_server.id
+}
+
+resource "aws_security_group_rule" "jaspersoft_server_ssh_ingress" {
+  type                     = "ingress"
+  from_port                = 22
+  to_port                  = 22
+  protocol                 = "tcp"
+  source_security_group_id = var.allow_ssh_from_sg_id
+  description              = "SSH from bastion"
+  security_group_id        = aws_security_group.jaspersoft_server.id
+}
+
+resource "aws_security_group_rule" "jaspersoft_server_http_ingress" {
+  type                     = "ingress"
+  from_port                = 8080
+  to_port                  = 8080
+  protocol                 = "tcp"
+  source_security_group_id = aws_security_group.alb.id
+  description              = "HTTP on 8080 from ALB"
+  security_group_id        = aws_security_group.jaspersoft_server.id
+}
+
+resource "aws_instance" "jaspersoft_server" {
+  ami           = "ami-00f499a80f4608e1b" #	eu-west-1 bionic 18.04 LTS amd64
+  instance_type = "t3.medium"
```

Comment:
Can you make this a variable?

Reply:
Yep, done, have made max memory for tomcat a variable too